backend/src/schema/resolvers/users.spec.js

```diff
@@ -568,3 +568,330 @@ describe('DeleteUser', () => {
     })
   })
 })
+
+
+
+describe('DeleteUser as Admin', () => {
+  const deleteUserMutation = gql`
+    mutation($id: ID!, $resource: [Deletable]) {
+      DeleteUser(id: $id, resource: $resource) {
+        id
+        name
+        about
+        deleted
+        contributions {
+          id
+          content
+          contentExcerpt
+          deleted
+          comments {
+            id
+            content
+            contentExcerpt
+            deleted
+          }
+        }
+        comments {
+          id
+          content
+          contentExcerpt
+          deleted
+        }
+      }
+    }
+  `
+  beforeEach(async () => {
+    variables = { id: ' u343', resource: [] }
+
+    user = await Factory.build('user', {
+      name: 'My name should be deleted',
+      about: 'along with my about',
+      id: 'u343',
+    })
+    await Factory.build(
+      'user',
+      {
+        id: 'not-my-account',
+      },
+      {
+        email: 'friends-account@example.org',
```

Comment:
```suggestion
        email: 'harassing-user@example.org',
```

maybe? not a big deal, but why is the `admin` deleting a `friends` account?